### PR TITLE
feat(lamp): add tabsBar

### PR DESCRIPTION
**Pull request by Doraemon91**
## Owner

- [x] Nickname(Tuya): 奥里
- [x] Intra-group review（组内审阅人员）： 若叶，萧瑟

## Description

tuya-panel-lamp-sdk add tabsBar

## Package

Please select the relevant package.

- [x] tuya-panel-lamp-sdk

## Type of change

Please select the relevant option.

- [x] New feature (non-breaking change which adds functionality)

## Self CheckList

Please check all options.

- [x] Intra-group review
- [x] ESLint lint
- [x] Example Run
- [x] Jest Run
- [x] No Grammer Error
- [x] No Spell Error
- [x] No Merge Conflict

### CODE REVIEW

File: example/src/i18n/strings.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ module.exports = {
```
Comment (XiaooSe) [inline example/src/i18n/strings.js:23]: 多语言文件在Lamp 文件夹下重新写吧，别用公共的多语言文件了

**Comment by XiaooSe** [comment]
LGTM

File: packages/tuya-panel-lamp-sdk/src/components/tabs_bar/interface.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  style: StyleProp<ViewStyle>;
+  isActive: boolean;
+  textStyle: StyleProp<TextStyle>;
+  activeTextStyle: StyleProp<TextStyle>;
+  horizontal: boolean;
+  onItemPress: () => void;
+  icon?: ReactNode;
+}
+
+export interface RadioBUttonProps {
```
Comment (paruki-gin) [inline packages/tuya-panel-lamp-sdk/src/components/tabs_bar/interface.ts:15]: Variable names should use camel case

File: packages/tuya-panel-lamp-sdk/src/components/tabs_bar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        res.push(node);
+        return res;
+      }, []);
+  };
+
+  return (
+    <View
+      onLayout={wrapperLayout}
+      pointerEvents={disabled ? 'none' : 'auto'}
+      style={[_wrapperStyle as [], { opacity: disabled ? disabledOpacity : 1 }]}
```
Comment (paruki-gin) [inline packages/tuya-panel-lamp-sdk/src/components/tabs_bar/index.tsx:181]: _wrapperStyle  requires a type definition

**Review by paruki-gin — COMMENTED** [review]
The component path and name need to be defined according to the specification

**Review by paruki-gin — APPROVED** [review]
lgtm

**Comment by youngjuning** [comment]
lgtm

**Pull request merged**